sympy/simplify/hyperexpand.py

```diff
@@ -1964,6 +1964,7 @@ def _hyperexpand(func, z, ops0=[], z0=Dummy('z0'), premult=1, prem=0,
     is multiplied by premult. Then ops0 is applied.
     premult must be a*z**prem for some a independent of z.
     """
+    ops0 = list(ops0)
```

Comment:
What happens if you skip this?

Reply:
Nothing, it seems. I ran several tests and all passed. I guess then I should delete this line in the next and the final commit before squashing.